topics/ui/principles/typography.md

```diff
@@ -49,71 +48,42 @@ Use the built-in text styles from the table below whenever possible.
   <td width="53%"> Usage Examples </td></tr>
 
   <tr>
-    <td> H0 bold <p><code>JBFont.h0().asBold()</code></p></td>
-    <td> Default + 12 </td>
-    <td>
-        Rich text headers. See the "What’s New" page example below.
-    </td>
-  </tr>
-
-  <tr>
-    <td> H1 bold <p><code>JBFont.h1().asBold()</code></p></td>
-    <td> Default + 9 </td>
-    <td> </td>
-  </tr>
-
-  <tr>
-    <td>H2 <p><code>JBFont.h2()</code></p></td>
-    <td> Default + 5 </td>
-    <td> </td>
-  </tr>
-
-  <tr>
-    <td> H2 bold <p><code>JBFont.h2().asBold()</code></p></td>
-    <td> </td>
+    <td> H1 <p><code>JBFont.h1().asBold()</code></p></td>
+    <td> Default +5 </td>
     <td>
-        Small page header. Examples: Plugin name, GitHub timeline header
+        Main page header. Example: Plugin name
     </td>
   </tr>
 
   <tr>
-    <td> H3 <p><code>JBFont.h3()</code></p></td>
-    <td> Default + 3 </td>
+    <td> H2 <p><code>JBFont.h2().asBold()</code></p></td>
+    <td> Default +3 </td>
     <td>
-        Accent body text
+        Small page header. Example: Headers in a plugin description
     </td>
   </tr>
 
   <tr>
-    <td>H3 bold <p><code>JBFont.h3().asBold()</code></p></td>
-    <td> </td>
-    <td>
-        Headers in dialogs with a small number of elements (Customize page on Welcome screen, Login page in Get from VCS dialog)
-    </td>
-  </tr>
-
-  <tr>
-    <td> H4 bold (Default bold) <p> <code>JBFont.h4().asBold()</code></p> <p> <code>JBFont.regular().asBold()</code></p> </td>
+    <td> Default <p><code>JBFont.regular()</code></p></td>
     <td> Default </td>
     <td>
-        <p>Header in dialogs with a large number of elements (Run configurations dialog), notification header, breadcrumbs in settings, header in navigation popup, accent elements in lists and trees.</p>
-        <p>Use <a href="group_header.md">Group header</a> to divide the page on groups, but not to draw too much attention to the headings.</p>
+        Main UI font. Examples: Labels, inputs, links, trees, tables and other controls; text outputs, notifications, shortcuts, group headers
```

Comment:
Possibly, leave only this part: "Main UI font. Examples: Labels, inputs, links, trees, tables and other controls". It's enough for an example